active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,282 @@
+
+| MaxColumns = 4 |
+|:--:|
+| ![alt text](multicolumn_example_1.png) |
+
+```xml
+<RadioButtons Header="Select Number" MaxComuns="4">
+    <x:String>1</x:String>
+    <x:String>2</x:String>
+    <x:String>3</x:String>
+    <x:String>4</x:String>
+</RadioButtons>
+```
+
+### Multiple columns (uncommon case)
+
+If you specify a ``MaxColumns`` value that is not equal to the number of items in the RadioButtons list, the list will arrange  itself in column-major order and put any uneven/remaining items in the first column.
+
+|MaxColumns = 3|
+|:--|
+|![alt text](multicolumn_example_2.png)|
+
+```xml
+<RadioButtons Header="Select Number" MaxColumns="3">
+    <x:String>1</x:String>
+    <x:String>2</x:String>
+    <x:String>3</x:String>
+    <x:String>4</x:String>
+    <x:String>5</x:String>
+    <x:String>6</x:String>
+    <x:String>7</x:String>
+</RadioButtons>
+```
+
+# Remarks
+The RadioButtons control has special navigation behavior that helps keyboard users navigate the list more quickly and more easily.
+
+## Putting focus into the RadioButtons list
+There are two states that the RadioButtons control can have with its RadioButton content:
+
+- A list of RadioButton controls where none are selected/checked
+- A list of RadioButton controls where one is already selected/checked
+
+The following two sections (below) cover the focus behavior when tabbing into the list in both cases outlined in the bullet list above.
+
+### A RadioButton is selected
+When a RadioButton is selected and the user tabs into the list, focus is put on the selected RadioButton.
+
+|Focus is on button<br>RadioButton group does not have focus| Tab has been pressed<br>RadioButton group now has focus|
+|:--:|:--:|
+| ![alt text](selecteditem_notabfocus.png) | ![alt text](selecteditem_tabfocus.png)|
+
+### No RadioButton is selected
+When no RadioButton controls are selected in the RadioButtons list, focus is put on the first RadioButton in the list.
+
+|Focus is on button<br>RadioButton group does not have focus | Tab has been pressed<br>RadioButton group now has focus|
+|:--:|:--:|
+| ![alt text](noselecteditem_notabfocus.png) | ![alt text](noselecteditem_tabfocus.png)|
+
+## Keyboard navigation
+When you have a single column list of RadioButton control options and you have already put focus onto an item, you can navigate the RadioButtons list in logical sequential order using the keyboard.
+
+- The up or down arrow keys will move to the "next" or "previous" logical item (defined in markup)
+- The left or right arrow keys will move spatially
+
+![alt text](keyboardnav_singlecol.png)
+
+This means that in purely vertical RadioButton group lists, left/right arrow keys will not do anything, but up/down will still navigate as expected. However, in purely horizontal RadioButton group lists, left and right will navigate the same way as up and down.
```

Comment:
What's the goal of this restriction? 
I tried this out and it feels weird to me that up-down navigates for a purely horizontal group but left-right does nothing for a purely vertical group. It's logically consistent as an implementation detail but as an end-user the behavior difference feels a bit weird.